src/com/rakovets/course/challenge/oop/classes/Point.java

```diff
@@ -0,0 +1,24 @@
+package com.rakovets.course.challenge.oop.classes;
+
+public class Point {
+	private double x;
+	private double y;
+
+	Point(double x, double y) {
+		this.x = x;
+		this.y = y;
+	}
+
+	double getX() {
+		return this.x;
+	}
+
+	double getY() {
+		return this.y;
+	}
+
+	double distance(Point point2) {
```

Comment:
Name for variable don't readable. Example:
```suggestion
	double distance(Point anotherPoint) {
```